app/townsquare/views.py

```diff
@@ -332,6 +332,11 @@ def get_following_tribes(request):
 
 
 def town_square(request):
+    data_results = JSONStore.objects.filter(view='results', key=None)
+    audience = '39102'
```

Comment:
dont you want to get the `.first()` result?

also would appreciate if this was cached in redis or similar so were not hitting the DB with this query every tTS pageview

Reply:
Done :)